packages/table/src/headers/columnHeaderCell.tsx

```diff
@@ -59,6 +62,15 @@ export interface IColumnHeaderCellProps extends IColumnNameProps, IProps {
     isColumnSelected?: boolean;
 
     /**
+     * If true, the column name (string or `ReactElement`) will be replaced with a fixed-height
```

Comment:
paren clause is not helpful, please remove. say "column `name`" to reference the prop, from which the user can infer the typings.